arm-sql/2014-04-01/swagger/serverCommunicationLinks.json

```diff
@@ -0,0 +1,308 @@
+          },
+          {
+            "name": "communicationLinkName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the server communication link."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ServerCommunicationLink"
+            },
+            "description": "The required parameters for creating a server communication link."
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/ServerCommunicationLink"
+            }
+          },
+          "202": {
+            "description": "Accepted"
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/communicationLinks": {
+      "get": {
+        "tags": [
+          "ServerCommunicationLinks"
+        ],
+        "operationId": "CommunicationLinks_ListByServer",
+        "description": "Gets a list of server communication links.",
+        "x-ms-examples": {
+          "List server communication links": { "$ref": "../examples/ServerCommunicationLinkList.json" }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK response definition.",
+            "schema": {
+              "$ref": "#/definitions/ServerCommunicationLinkListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ServerCommunicationLinkProperties": {
+      "properties": {
+        "state": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The state."
+        },
+        "partnerServer": {
+          "type": "string",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ],
+          "description": "The name of the partner server."
+        }
+      },
+      "required": ["partnerServer"],
+      "description": "The properties of an Server communication link."
+    },
+    "ServerCommunicationLink": {
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/ServerCommunicationLinkProperties",
+          "description": "The properties of resource.",
+          "x-ms-client-flatten": true
+        },
+        "location":{
```

Comment:
Do tracked resources always have tags? This resource didn't have them in fiddler.  And location wasn't required for resource create, which I believe it is for tracked resources.